packages/next/build/webpack-config.ts

```diff
@@ -291,6 +294,15 @@ export default async function getBaseWebpackConfig(
       ? 'anonymous'
       : config.crossOrigin
 
+  let customAppFile = await findPageFile(
```

Comment:
```suggestion
  let customAppFile
  if (config.experimental.css) {
    customAppFile = await findPageFile(
      path.join(dir, 'pages'),
      '/_app',
      config.pageExtensions
    )
    if (customAppFile) {
      customAppFile = path.resolve(path.join(dir, 'pages', customAppFile))
    }
  }
```
might be good to have this look up behind the experimental flag also since it's not needed otherwise